caffe2/contrib/ideep/CMakeLists.txt

```diff
@@ -1,20 +1,20 @@
-if(USE_MKL AND USE_IDEEP AND CAFFE2_COMPILER_SUPPORTS_AVX2_EXTENSIONS)
+if(CAFFE2_USE_IDEEP)
   message(STATUS "Including IDEEP operators")
 
   # ---[ CPU files.
-  file(GLOB_RECURSE avx2_srcs *.cc)
+  file(GLOB_RECURSE tmp *.cc)
+  set(Caffe2_CPU_SRCS ${Caffe2_CPU_SRCS} ${tmp})
   # exclude test files and gpu files
   file(GLOB_RECURSE tmp *_test.cc)
-  exclude(avx2_srcs "${avx2_srcs}" ${tmp})
+  exclude(Caffe2_CPU_SRCS "${Caffe2_CPU_SRCS}" ${tmp})
 
-  add_library(Caffe2_ideep_operators OBJECT ${avx2_srcs})
-  add_dependencies(Caffe2_ideep_operators Caffe2_PROTO)
-  set_target_properties(Caffe2_ideep_operators PROPERTIES COMPILE_FLAGS "-mavx2")
```

Comment:
We don't need `-mavx2` anymore?

Reply:
Yes. decide to remove it to make more clean.